integration-tests/features/steps/common.py

```diff
@@ -867,7 +896,7 @@ def check_licenses(node, expected_licenses):
 
 
 @then('I should find the following licenses ({licenses}) under the path {path}')
-def stack_analysis_check_licenses(json_data):
+def stack_analysis_check_licenses(context, json_data):
     licenses = split_comma_separated_list(licenses)
```

Comment:
It's been reworked here: https://github.com/fabric8-analytics/fabric8-analytics-common/pull/192/files